api/external.go

```diff
@@ -381,16 +381,22 @@ func (a *API) Provider(ctx context.Context, name string, scopes string) (provide
 		return provider.NewGitlabProvider(config.External.Gitlab, scopes)
 	case "google":
 		return provider.NewGoogleProvider(config.External.Google, scopes)
+	case "linkedin":
+		return provider.NewLinkedInProvider(config.External.Google, scopes)
```

Comment:
Maybe it should be Linkedin instead of Google ?

Reply:
Yeah, will fix now.